jax/_src/util.py

```diff
@@ -332,7 +332,7 @@ def tuple_delete(t, idx):
 
 def tuple_replace(t, idx, val):
   assert 0 <= idx < len(t), (idx, len(t))
-  return t[:idx] + (val,) + t[idx:]
+  return t[:idx] + (val,) + t[idx+1:]
```

Comment:
In general, you should follow the style of the surrounding code. Put spaces around binary operators like `+`.

Reply:
Thanks for the feedback! 

Best wishes
Josh

Reply:
Hi Josh, would you like to make the change and close this PR? thanks!